src/main/java/com/gooddata/md/ScheduledMail.java

```diff
@@ -3,14 +3,9 @@
  */
 package com.gooddata.md;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
I don't know about any agreement. And there are also usages of star imports in other classes, so lets resolve it later once we have some agreement.